README.md

```diff
@@ -38,8 +40,10 @@ Next, create queues:
 `finalized-products`  
 `update-products`
 
-Create exchanges:  
+// REVIEW-VINI: Maybe add these steps as title, like: 
+### Create exchanges:
 
+// REVIEW-VINI: Do we need to create a bind for this queue?
```

Comment:
this particular exchange Quarkus will create automatically, so it's no need to create manually, good point